charts/loki-simple-scalable/values.yaml

```
@@ -69,6 +69,9 @@ loki:
     server:
       http_listen_port: 3100
       grpc_listen_port: 9095
+      {{- if .Values.loki.extraServerConfig }}
+      {{- toYaml .Values.loki.extraServerConfig | nindent 2 }}
+      {{- end }}
```

I would say `with` statement should be better:

```suggestion
      {{- with .Values.loki.extraServerConfig }}
      {{- toYaml . | nindent 2 }}
      {{- end }}
```

The difference between `if` and `with` is while given `{}`, `if` will render as `{}`, whereas `with` will skip. You could try comparing these two with `helm template lokiss .`

---

Indeed, thanks for noting this @zanhsieh . Changes pushed!